tests/test_bench.py

```diff
@@ -688,10 +677,10 @@ def test_get_benchmark_faulty_parameters() -> None:
         get_benchmark("wrong_name", 2, 6)
     match = "Selected level must be in"
     with pytest.raises(ValueError, match=match):
-        get_benchmark(
+        get_benchmark(  # type: ignore[call-overload]
```

Comment:
Yes it is, since I use wrong parameter types to test whether `get_benchmark` returns the assumed `ValueError`. Or do you know any other way around it, if, e.g., an `int` is used instead of an expected `str`?

Reply:
Ah. Didn't realise these are cases checking for wrong inputs.
Technically one could think about removing those and solely relying on the type system to warn the user. But that's something for the future.
Feel free to resolve here.